Sources/SwiftInspectorCore/InitializerAnalyzer.swift

```diff
@@ -124,7 +129,25 @@ public struct InitializerStatement: Equatable {
 
   public struct Parameter: Equatable {
     public let name: String
-    public let typeName: String
+    public let typeNames: [String]
+
+    /// The most common use case is for a parameter to only contain one type
+    /// That's why we have this computed property as a convenience
+    public var typeName: String? {
```

Comment:
Do we use this anywhere?

Reply:
Yup! On tests https://github.com/fdiaz/SwiftInspector/blob/f3e16254429cc08033b07157ac5274efe3b25d4e/Sources/SwiftInspectorCore/Tests/InitializerAnalyzerSpec.swift#L75